modules/ROOT/pages/configure-hardening.adoc

```diff
@@ -0,0 +1,23 @@
+= Configure Environment Hardening on Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+By design, Runtime Fabric automatically provides multiple aspects of environment hardening for your Mule installation. 
+For example, by leveraging Docker and Kubernetes, app isolation is achieved by default.  
```

Comment:
and by following containerization best practices, isolation at the Mule runtime and application level is achieved by default.

Reply:
Got it - please make sure I added what was intended - thank you!